src/porepy/numerics/fv/biot.py

```diff
@@ -394,13 +366,27 @@ def discretize(self, sd: pp.Grid, sd_data: dict) -> None:
         # then the discretization does quite a few different things
         active_stress = sps.csr_matrix((nf * nd, nc * nd))
         active_bound_stress = sps.csr_matrix((nf * nd, nf * nd))
-        active_grad_p = sps.csr_matrix((nf * nd, nc))
-        active_div_u = sps.csr_matrix((nc, nc * nd))
-        active_bound_div_u = sps.csr_matrix((nc, nf * nd))
-        active_stabilization = sps.csr_matrix((nc, nc))
+
         active_bound_displacement_cell = sps.csr_matrix((nf * nd, nc * nd))
         active_bound_displacement_face = sps.csr_matrix((nf * nd, nf * nd))
-        active_bound_displacement_pressure = sps.csr_matrix((nf * nd, nc))
+
+        active_scalar_gradient, active_displacement_divergence, active_bound_displacement_divergence, active_stabilization = (
+            {},
+            {},
+            {},
+            {},
+        )
+        active_bound_displacement_pressure = {}
```

Comment:
Please choose between defining all active_... with one statement and on individual lines.